src/Compilers/CSharp/Portable/Errors/ErrorFacts.cs

```diff
@@ -573,1877 +573,1875 @@ internal static int GetWarningLevel(ErrorCode code)
-            }
+                ErrorCode.WRN_ALinkWarn
+                or ErrorCode.WRN_UnreferencedField
+                or ErrorCode.WRN_UnreferencedFieldAssg
+                or ErrorCode.WRN_UnreferencedEvent
+                or ErrorCode.WRN_UnassignedInternalField
+                or ErrorCode.ERR_MissingPredefinedMember
+                or ErrorCode.ERR_PredefinedTypeNotFound
+                or ErrorCode.ERR_NoEntryPoint
+                or ErrorCode.WRN_InvalidMainSig
+                or ErrorCode.ERR_MultipleEntryPoints
+                or ErrorCode.WRN_MainIgnored
+                or ErrorCode.ERR_MainClassNotClass
+                or ErrorCode.WRN_MainCantBeGeneric
+                or ErrorCode.ERR_NoMainInClass
+                or ErrorCode.ERR_MainClassNotFound
+                or ErrorCode.WRN_SyncAndAsyncEntryPoints
+                or ErrorCode.ERR_BadDelegateConstructor
+                or ErrorCode.ERR_InsufficientStack
+                or ErrorCode.ERR_ModuleEmitFailure
+                or ErrorCode.ERR_TooManyLocals
+                or ErrorCode.ERR_BindToBogus
+                or ErrorCode.ERR_ExportedTypeConflictsWithDeclaration
+                or ErrorCode.ERR_ForwardedTypeConflictsWithDeclaration
+                or ErrorCode.ERR_ExportedTypesConflict
+                or ErrorCode.ERR_ForwardedTypeConflictsWithExportedType
+                or ErrorCode.ERR_ByRefTypeAndAwait
+                or ErrorCode.ERR_RefReturningCallAndAwait
+                or ErrorCode.ERR_SpecialByRefInLambda
+                or ErrorCode.ERR_DynamicRequiredTypesMissing
+                or ErrorCode.ERR_CannotBeConvertedToUtf8
+                or ErrorCode.ERR_FileTypeNonUniquePath
+                or ErrorCode.ERR_InterceptorSignatureMismatch
+                or ErrorCode.ERR_InterceptorMustHaveMatchingThisParameter
+                or ErrorCode.ERR_InterceptorMustNotHaveThisParameter
+                or ErrorCode.ERR_DuplicateInterceptor
+                or ErrorCode.WRN_InterceptorSignatureMismatch
+                or ErrorCode.ERR_InterceptorNotAccessible
+                or ErrorCode.ERR_InterceptorScopedMismatch
+                or ErrorCode.WRN_NullabilityMismatchInReturnTypeOnInterceptor
+                or ErrorCode.WRN_NullabilityMismatchInParameterTypeOnInterceptor
+                or ErrorCode.ERR_InterceptorCannotInterceptNameof
+                or ErrorCode.ERR_SymbolDefinedInAssembly
+                or ErrorCode.ERR_InterceptorArityNotCompatible
+                or ErrorCode.ERR_InterceptorCannotBeGeneric
+                or ErrorCode.ERR_InterceptableMethodMustBeOrdinary
+                or ErrorCode.ERR_PossibleAsyncIteratorWithoutYield
+                or ErrorCode.ERR_PossibleAsyncIteratorWithoutYieldOrAwait
+                    => true, // Update src\EditorFeatures\CSharp\LanguageServer\CSharpLspBuildOnlyDiagnostics.cs
+                             // whenever new values are added here.
```

Comment:
nit: I subjectively find the below suggestion more readable, use it or don't at your discrection.

```suggestion
                    // Update src\EditorFeatures\CSharp\LanguageServer\CSharpLspBuildOnlyDiagnostics.cs
                    // whenever new values are added here.
                    => true,
```